app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_if_inputs_values_are_zeros() -> None:
+    assert get_human_age(0, 0) == [0, 0], "False"
+
+
+def test_inputs_smaller_then_default_values() -> None:
+    assert get_human_age(14, 14) == [0, 0], "False"
```

Comment:
The test name `test_inputs_smaller_then_default_values` is misleading. The values 14, 14 are not smaller than default values if the default is considered to be 0. Consider renaming the test to better reflect the scenario being tested.